osgp/protocol-adapter-oslp/web-device-simulator/src/main/java/org/opensmartgridplatform/webdevicesimulator/service/OslpChannelHandler.java

```diff
@@ -1122,6 +1039,25 @@ private void handleSetTransitionRequest(final Device device, final SetTransition
         }
     }
 
+    private void sendEvent(final Device device, final Oslp.Event event, final String description) {
+        this.sendEventWithCustomDelay(device, event, description, 3000);
+    }
+
+    private void sendEventWithCustomDelay(final Device device, final Oslp.Event event, final String description,
+            final int delay) {
+        // Send an event.
+        final Timer timer = new Timer();
+        timer.schedule(new TimerTask() {
+
+            @Override
+            public void run() {
+                OslpChannelHandler.this.registerDevice.sendEventNotificationCommand(device.getId(), event.getNumber(),
+                        description, null);
+            }
+
+        }, delay);
+    }
+
     //@formatter:off
     /*
 osgp_core=# select * from event where device = (select id from device where device_identification = 'ELS-108600000502016');
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

Reply:
Under the boy scouting rules (leave your environment cleaner than you found it) you could consider removing this block.
It is unclear to me why it is there and how it is useful, and removing it would get rid of a "major" sonar issue.

As an aside, I was wondering about something else. The formatter off and on comments and whether they are needed.
I know that (at least in Eclipse) you can start the multi-line comment with `/*-` instead of `/*` to prevent autoformatting to change the contents. Is this very IDE specific or is it more general and does IntelliJ handle it in the same way?

Reply:
I moved this piece of information to a text file in the resources folder. This will resolve the SonarQube issue.